styles/charts/_layout.scss

```diff
@@ -117,11 +120,11 @@ $chart-title-font-size: 1.143em !default;
     }
 
     .k-leftHandle {
-        left: -4px;
+        left: -12px;
     }
 
     .k-rightHandle {
-        right: -4px;
+        right: -12px;
```

Comment:
These seem to be related to the handle width. Use a variable to show the dependency.